name: Status check functions
on:
  push:
    paths: "status-check-functions.yml"
jobs:
  fail:
    runs-on: ubuntu-latest
    steps:
     # - run: exit 1          #終了ステータスが0以外なので、エラーが発生する
      - run: echo "Hello"
        if: ${{ failure() }}  #エラーが発生しても、failure関数があるので実行される
